policyengine_us/variables/gov/states/co/tax/income/credits/eitc/co_eitc.py

```diff
@@ -13,4 +13,5 @@ class co_eitc(Variable):
     def formula(tax_unit, period, parameters):
         federal_eitc = tax_unit("eitc", period)
         p = parameters(period).gov.states.co.tax.income.credits
-        return federal_eitc * p.eitc.match
+        estimated_adjusted_factor = p.eitc.estimated_adjusted_factor
+        return federal_eitc * p.eitc.match.calc(estimated_adjusted_factor)
```

Comment:
isnt the factor just a value? We would just need a "*"

Reply:
But we need to match the correct percentage according to the factor? so I use the .cal() here

Reply:
My bad I meant, why do we make the match percentage dependent on the adjustment factor? Dont we just multiply it by the factor and the match is based on income?

Reply:
It's based on this rule I think, if I understand it correctly.
<img width="738" alt="image" src="https://github.com/user-attachments/assets/ee5d2e2f-a944-4f11-9917-151a8d1e8cb5">